src/cosasTest.wtest

```diff
@@ -0,0 +1,188 @@
+import cosas.*
+
+describe "test de cosas" {
+
+//para hacer mas sencillo el assert de cada test 
+//pero es raro no ver el assert en el test y da alerta Wollok
+const cosaPesoYPeligrosidadSon = { unaCosa, unPeso, unaPeligrosidad => assert.that(unaCosa.peso() == unPeso and unaCosa.peligrosidad() == unaPeligrosidad)}

```

Comment:
Está buena la idea, qué pena que Wollok se queja de eso, habría que reportarlo para ver si se puede mejorar.
Sin embargo, yo cambiaría el assert.that por 2 assert.equals, así queda más claro en caso de error.

Reply:
Hola, sí, repito el problema de el equals!!! Disculpas. Gracias a la corrección ahora lo voy a tener más en cuenta!